app/celery/tasks/emails.py

```diff
@@ -0,0 +1,30 @@
+from app.clients.example_email_client import ExampleEmailClient
+from app.db.session import SessionLocal
+from app.main import celery
+from app.schemas.pagination_schema import ListFilter
+from app.services.emails_service import EmailService
+from app.services.users_service import UsersService
+
+from app.repositories.users_repository import users_repository
+
+
+@celery.task
+def send_reminder_email() -> None:
+    session = SessionLocal()
+    try:
+        users = UsersService(session, users_repository).list(ListFilter())
```

Comment:
Why empty list filter?

Reply:
Is mandatory to pass it and here is just an example. Maybe i can add explicit page and page_size so it is clearer